cssom-1/Overview.bs

```diff
@@ -2190,28 +2192,51 @@ The <dfn method for=CSSStyleDeclaration>setProperty(<var>property</var>, <var>va
+   being the <a for="CSSStyleDeclaration">declarations</a>.
+   <li>If <var>longhand result</var> is true, let <var>updated</var> be true.
+  </ol>
+ <li>Otherwise, let <var>updated</var> be the result of <a lt="set a CSS declaration">set the CSS declaration</a>
+ <var>property</var> with value <var>component value list</var>, with the <i>important</i> flag set if
+ <var>priority</var> is not the empty string, and unset otherwise, and with the list of declarations being the
  <a for="CSSStyleDeclaration">declarations</a>.
- <li><a>Update style attribute for</a> the <a>CSS declaration block</a>.
+ <li>If <var>updated</var> is true, <a>update style attribute for</a> the <a>CSS declaration block</a>.
 </ol>
 
 To <dfn>set a CSS declaration</dfn> <var>property</var> with a value <var>component value list</var> and optionally with an <i>important</i> flag set, in
-a list of declarations <var>declarations</var>, follow these steps:
+a list of declarations <var>declarations</var>, the user agent must ensure the following constraints hold after its steps:
 
-<ol>
- <li>If <var>property</var> is a <a>case-sensitive</a> match for a <a for="CSS declaration">property
- name</a> of a <a>CSS declaration</a> in <var>declarations</var>, remove that <a>CSS declaration</a>.
- <li>Append a new <a>CSS declaration</a> with the <a for="CSS declaration">property name</a> <var>property</var>
- to <var>declarations</var> and let <var>declaration</var> be that <a>CSS declaration</a>.
- <li>Set <var>declaration</var>'s <a for="CSS declaration">value</a> to <var>component value list</var>.
- <li>If the <i>important</i> flag is set, set <var>declaration</var>'s <a for="CSS declaration">important flag</a>.
- Otherwise, unset <var>declaration</var>'s <a for="CSS declaration">important flag</a>.
-</ol>
+<ul>
+ <li>
+  Exactly one <a>CSS declaration</a> whose <a for="CSS declaration">property name</a> is a
+  <a>case-sensitive</a> match of <var>property</var> must exist in <var>declarations</var> with value
+  being <var>component value list</var>. The <a for="CSS declaration">important flag</a> of this
+  <a>CSS declaration</a> must be <a>set</a> if <i>important</i> flag is set, and <a>unset</a> otherwise.
+  Such declaration is referenced as the <var>target declaration</var> below.
+
+ <li>
+  Any <a>CSS declaration</a> which is not the <var>target declaration</var> must not be changed, inserted,
+  or removed from <var>declarations</var>.
+
+ <li>
+  If there are <a>CSS declaration</a>s in <var>declarations</var> whose
+  <a for="CSS declaration">property name</a> is in the same <a>logical property group</a> as
+  <var>property</var>, but has a different <a>logical kind</a>, <var>target declaration</var> must be
+  at an index after all of those <a>CSS declaration</a>s.
+
+ <li>
+  The steps must return true if the serialization of <var>declarations</var> is changed as a result of
```

Comment:
Word this as "If the serialization of declarations was changed as result of these steps, then return true. Otherwise, return false."

Reply:
This wording "If ... then ... otherwise" sounds like part of steps, but this should be a constraint I think.